ui/nuxeo-data-table/data-table-cell.js

```diff
@@ -28,6 +28,16 @@ import './data-table-templatizer-behavior.js';
           :host([hidden]) {
             display: none;
           }
+
+          :host([actions='single']) {
+            flex: 0 0 80px;
+            padding: 0;
+          }
+
+          :host([actions='double']) {
```

Comment:
This is a generic element with logic specific to actions. Since we're already setting `style` attributes directly on the data table, I'd be in favor or computing the styles of the cell and actions elements (or their containers) directly on the data table. This would probably required the actions element to have `--layout-flex`.

Reply:
You're right. Please checkout the new solution.